src/constants.json

```diff
@@ -1,3 +1,10 @@
 {
-  "version": "0.1.3"
+  "livenet": {
+    "hub": "https://hub.snapshot.org",
```

Comment:
I get `snapshot.min.js:15          POST https://hub.snapshot.org/ 404` error when i try to vote @bonustrack

Reply:
I use same code as here https://docs.snapshot.org/snapshot.js#init-client